src/workers/newNotificationV2Mail.ts

```diff
@@ -637,6 +638,9 @@ const notificationToTemplateData: Record<NotificationType, TemplateDataFunc> = {
       },
     };
   },
+  source_post_added: async () => {
+    return null;
+  },
```

Comment:
Email notification is currently out of scope.

Reply:
Are you sure email is out of scope? It's weird we should include it

Reply:
Yes, it was mentioned in the spec and confirmed [here](https://dailydotdev.atlassian.net/wiki/spaces/HAN/pages/801374248/Subscribe+to+source?focusedCommentId=818151448)

Reply:
Ok, I asked for email. Without an email it will have almost no impact